upstream-community-operators/synopsys/synopsys-operator.2019.4.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,479 @@
+   - name: OpsSight Data Sheet
+     url: https://www.synopsys.com/content/dam/synopsys/sig-assets/datasheets/openshift-ds-ul.pdf
+   - name: Black Duck Alert Documentation
+     url: https://synopsys.atlassian.net/wiki/spaces/INTDOCS/pages/622882/Black+Duck+Alert
+   - name: Synopsys Operator Source Code
+     url: https://github.com/blackducksoftware/synopsys-operator
+  installModes:
+   - type: OwnNamespace
+     supported: true
+   - type: SingleNamespace
+     supported: false
+   - type: MultiNamespace
+     supported: false
+   - type: AllNamespaces
+     supported: true
+  install:
+    strategy: Deployment
+    spec:
+      deployments:
+        - name: synopsys-operator
+          spec:
+            replicas: 1
+            selector:
+              app: synopsys-operator
+              component: operator
+            template:
+              metadata:
+                labels:
+                  app: synopsys-operator
+                  component: operator
+                name: synopsys-operator
+                namespace: placeholder
+              spec:
+                containers:
+                - args:
+                  - /etc/synopsys-operator/config.json
+                  command:
+                  - ./operator
+                  env:
+                  - name: SEAL_KEY
+                    valueFrom:
+                      secretKeyRef:
+                        key: SEAL_KEY
+                        name: blackduck-secret
+                        optional: false
+                  image: docker.io/blackducksoftware/synopsys-operator:2019.4.0
+                  imagePullPolicy: Always
+                  name: synopsys-operator
+                  ports:
+                  - containerPort: 8080
+                    protocol: TCP
+                  resources: {}
+                  terminationMessagePath: /dev/termination-log
+                  terminationMessagePolicy: File
+                  volumeMounts:
+                  - mountPath: /etc/synopsys-operator
+                    name: synopsys-operator
+                  - mountPath: /opt/synopsys-operator/tls
+                    name: synopsys-operator-tls
+                - command:
+                  - ./app
+                  env:
+                  - name: ADDR
+                    value: 0.0.0.0
+                  - name: PORT
+                    value: "3000"
+                  - name: GO_ENV
+                    value: production
+                  image: docker.io/blackducksoftware/synopsys-operator:2019.4.0
+                  imagePullPolicy: Always
+                  name: synopsys-operator-ui
+                  ports:
+                  - containerPort: 3000
+                    protocol: TCP
+                  resources: {}
+                  terminationMessagePath: /dev/termination-log
+                  terminationMessagePolicy: File
+                dnsPolicy: ClusterFirst
```

Comment:
I believe this is misaligned. Please confirm that your deployment is valid

Reply:
apologies, you are correct. fixed locally.